web_src/js/features/notification.js

```diff
@@ -163,58 +145,51 @@ async function updateNotificationCountWithCallback(callback, timeout, lastCount)
 async function updateNotificationTable() {
   const notificationDiv = $('#notification_div');
   if (notificationDiv.length > 0) {
-    const data = await $.ajax({
-      type: 'GET',
-      url: `${appSubUrl}/notifications${window.location.search}`,
-      data: {
+    try {
+      const params = new URLSearchParams({
         'div-only': true,
         'sequence-number': ++notificationSequenceNumber,
+      });
+      const url = `${appSubUrl}/notifications${window.location.search}${window.location.search ? '&' : '?'}${params.toString()}`;
```

Comment:
```suggestion
      const params = new URLSearchParams(window.location.search);
      params.set('div-only', true)
      params.set('sequence-number', ++notificationSequenceNumber)
      const url = `${appSubUrl}/notifications${params.toString()}`;
```

Seems cleaner and should work as long as there are no duplicate params, please try it.

Reply:
Yeah it works, I just had to add a `?`
https://github.com/go-gitea/gitea/pull/29817/commits/c815ff0720603a57468fc47fd99dd79a9cc0c31e

Reply:
Right, I forgot `?` is not part of URLSearchParams string form.